docs/release-notes/2.16.0.asciidoc

```diff
@@ -0,0 +1,62 @@
+:issue: https://github.com/elastic/cloud-on-k8s/issues/
+:pull: https://github.com/elastic/cloud-on-k8s/pull/
+
+[[release-notes-2.16.0]]
+== {n} version 2.16.0
+
+
+
+[[feature-2.16.0]]
+[float]
+=== New features
+
+* Support remote clusters using API keys {pull}8089[#8089] (issues: {issue}7818[#7818], {issue}8167[#8167])
+
+[[enhancement-2.16.0]]
+[float]
+=== Enhancements
+
+* Make helm charts consistent with how fields in spec are handled in eck-fleet-server. {pull}8285[#8285]
```

Comment:
```suggestion
* Make Helm charts consistent in how fields in spec are handled (eck-fleet-server). {pull}8285[#8285]
```
Capital initial for Helm: I am sure we should do that for consistency
The rewording, not 100% as I am no native speaker, but something in the original sentence is a bit odd.

Reply:
> something in the original sentence is a bit odd.

I don't disagree. I've attempted a re-wording of this. Lmk your thoughts @pebrc